server/fishtest/__init__.py

```diff
@@ -23,15 +24,25 @@ def main(global_config, **settings):
     config.include("pyramid_mako")
     config.set_default_csrf_options(require_csrf=False)
 
-    def static_file_hash(filepath):
-        full_path = Path(__file__).parent / "./static/{}".format(filepath)
-        with open(full_path, "r") as f:
+    def static_file_full_path(static_file_path):
+        return Path(__file__).parent / "./static/{}".format(static_file_path)
+
+    def static_file_hash(static_file_path):
+        with open(static_file_full_path(static_file_path), "r") as f:
             return hashlib.md5(f.read().encode("utf-8")).hexdigest()
 
+    # the same hash calculated by browser for sub-resource integrity checks:
+    # https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity
+    dark_theme_sha256_integrity = subprocess.run(
+        "sha256sum {} | awk '{{ print $1 }}' | xxd -r -p | base64".format(
```

Comment:
`"openssl {} dgst -sha256 -binary | openssl base64 -A".format(`

this Mozilla suggestion uses only `openssl` (simpler to add the requirement in the server setup script), already tested working with your code